packages/ddp-client/common/livedata_connection.js

```diff
@@ -1708,6 +1725,19 @@ export class Connection {
       return true;
     });
 
+    if (!self.shouldRetryMethods) {
+        self.shouldRetryMethods = true;
+        // Run the callback on all remaining method calls with the error as the parameter
+        if (isEmpty(oldOutstandingMethodBlocks)) return;
```

Comment:
There is no point for doing an extra `isEmpty()` check here, the loop below already effectively checks that (the condition will be immediately `false`).

Reply:
fixed